editor/import/resource_importer_dynamic_font.cpp

```diff
@@ -48,7 +46,6 @@ String ResourceImporterDynamicFont::get_visible_name() const {
 
 void ResourceImporterDynamicFont::get_recognized_extensions(List<String> *p_extensions) const {
 	if (p_extensions) {
-#ifdef MODULE_FREETYPE_ENABLED
```

Comment:
CC @bruvzg - is this correct?

This is editor code alright, so freetype is included, but the project may be exported to a build without freetype. Will these imported dynamic fonts work then at runtime?

Reply:
In second thought, this will always be true in the editor, so regardless of what export templates are configured with, these extensions will always be considered supported in the editor.

If that actually causes issues when exporting a project with TTFs files with a no-freetype export template, it would require a different fix anyway.